WalletWasabi.Gui/Controls/WalletExplorer/CoinListView.xaml

```diff
@@ -48,23 +49,24 @@
         <controls:SortingArrow Grid.Column="5" Text="Clusters" Command="{Binding SortCommand}" SortDirection="{Binding ClustersSortDirection}" />
       </Grid>
       <StackPanel Margin="0 10 0 0" Spacing="10" DockPanel.Dock="Bottom" Orientation="Horizontal">
-        <CheckBox Content = "Select All" IsThreeState="True" IsChecked = "{Binding SelectAllCheckBoxState}" Command="{Binding SelectAllCheckBoxCommand}" />
-        <CheckBox Content = "Select All Private" IsThreeState="True" IsChecked = "{Binding SelectPrivateCheckBoxState}" Command="{Binding SelectPrivateCheckBoxCommand}" ToolTip.Tip="Coins that have acquired the privacy target." />
-        <CheckBox Content = "Select All Non-Private" IsThreeState="True" IsChecked = "{Binding SelectNonPrivateCheckBoxState}" Command="{Binding SelectNonPrivateCheckBoxCommand}" ToolTip.Tip="Coins that have not acquired the privacy target." />
+        <CheckBox Content="Select All" IsThreeState="True" IsChecked="{Binding SelectAllCheckBoxState}" Command="{Binding SelectAllCheckBoxCommand}" />
+        <CheckBox Content="Select All Private" IsThreeState="True" IsChecked="{Binding SelectPrivateCheckBoxState}" Command="{Binding SelectPrivateCheckBoxCommand}" ToolTip.Tip="Coins those acquired the strongest privacy shield." />
+        <CheckBox Content="Select All Non-Private" IsThreeState="True" IsChecked="{Binding SelectNonPrivateCheckBoxState}" Command="{Binding SelectNonPrivateCheckBoxCommand}" ToolTip.Tip="Coins those do not have the strongest privacy shield." />
         <StackPanel Spacing="10" DockPanel.Dock="Bottom" Orientation="Horizontal" IsVisible="{Binding IsAnyCoinSelected}">
-          <TextBlock Text = "|" />
-          <TextBlock Text = "Selected Amount:" />
-          <TextBlock Foreground="YellowGreen" Text = "{Binding SelectedAmount, ConverterParameter=8, Converter={StaticResource LurkingWifeModeStringConverter}}" />
-          <TextBlock Text = "BTC" />
-          <TextBlock Text ="Merging unmixed coins with mixed ones undo those mixes." Classes="warningMessage" IsVisible="{Binding LabelExposeCommonOwnershipWarning}" />
+          <TextBlock Text="|" />
+          <TextBlock Text="Selected Amount:" />
+          <TextBlock Foreground="YellowGreen" Text="{Binding SelectedAmount, ConverterParameter=8, Converter={StaticResource LurkingWifeModeStringConverter}}" />
+          <TextBlock Text="BTC" />
+          <TextBlock Text="Merging unmixed coins with mixed ones undo those mixes." Classes="warningMessage" IsVisible="{Binding LabelExposeCommonOwnershipWarning}" />
         </StackPanel>
+        <controls:BusyIndicator IsBusy="{Binding FiltersLeft, Converter={StaticResource ShouldDisplayValueConverter}}" Text="{Binding FiltersLeft, StringFormat=Downloading \{0\} filters\, your coin list may be incorrect... }"/>
       </StackPanel>
```

Comment:
I get this: 
![image](https://user-images.githubusercontent.com/9844978/61622739-a9b6ee80-ac75-11e9-9f97-3d9bdb13e0f2.png)
I don't know the solution yet. Horizontal StackPanel should have a limitation and start a new line if there are too much controls in it.

Reply:
Mhhh - looks rather clustered here...
How about this is moved to the bottom status bar?